_RFCs/0001-rfc-for-rfcs.md

```diff
@@ -0,0 +1,290 @@
+    needs to have already been clearly articulated, and there should not be a
+    strong consensus *against* that position outside of the maintainers. We will
+    use our best judgment in taking this step, and the FCP itself
+    ensures there is ample time and notification for stakeholders to push back
+    if it is made prematurely.
+    - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
+    a *summary comment* trying to lay out the current state of the discussion
+    and major tradeoffs/points of disagreement.
+    - Before actually entering FCP, *all* Core-Plans Maintainers must sign off;
+    this is often the point at which many maintainers first review the RFC
+    in full depth.
+  - The FCP lasts ten calendar days, so that it is open for at least 5 business
+    days. It is also advertised widely,
+    e.g. in [ the #announcements channel in Slack](habitat-sh.slack.com). This way all
+    stakeholders have a chance to lodge any final objections before a decision
+    is reached.
+  - In most cases, the FCP period is quiet, and the RFC is either merged or
+    closed. However, sometimes substantial new arguments or ideas are raised,
+    the FCP is canceled, and the RFC goes back into development mode.
+
+## The RFC life-cycle
+[The RFC life-cycle]: #the-rfc-life-cycle
+
+Once an RFC becomes "active" then authors may implement it and submit the
+change as a pull request to the Core-Plans repo. Being "active" is not a rubber
+stamp, and in particular still does not mean the change will ultimately be
+merged; it does mean that in principle all the major stakeholders have agreed
+to the change and are amenable to merging it.
+
+Furthermore, the fact that a given RFC has been accepted and is "active"
+implies nothing about what priority is assigned to its implementation, nor does
+it imply anything about whether a Core-Plans developer has been assigned the task of
+implementing the change. While it is not *necessary* that the author of the
+RFC also write the implementation, it is by far the most effective way to see
+an RFC through to completion: authors should not expect that other project
+developers will take on responsibility for implementing their accepted feature.
+
+Modifications to "active" RFCs can be done in follow-up pull requests. We
+strive to write each RFC in a manner that it will reflect the final design of
+the feature; but the nature of the process means that we cannot expect every
+merged RFC to actually reflect what the end result will be.
+
+In general, once accepted, RFCs should not be substantially changed. Only very
+minor changes should be submitted as amendments. More substantial changes
+should be new RFCs, with a note added to the original RFC. Exactly what counts
+as a "very minor change" is up to the core-plans maintainers to decide.
+
+## Reviewing RFCs
+[Reviewing RFCs]: #reviewing-rfcs
+
+While the RFC pull request is up, the assigned shepherd may schedule meetings with the
```

Comment:
This is the first use of `shepherd` - is this a term that needs explanation or is there an existing conceptual term here?